internals/controllers/capp_controller.go

```diff
@@ -42,31 +43,33 @@ type CappReconciler struct {
 // +kubebuilder:rbac:groups=route.openshift.io,resources=routes,verbs=get;list;watch;update;create
 
 func (r *CappReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Log = log.FromContext(ctx)
+	logger := log.FromContext(ctx).WithName("container-app-controller").WithValues("Capp", req.NamespacedName)
+	logger.Info("starting Reconcile")
 	capp := rcsv1alpha1.Capp{}
 	if err := r.Client.Get(ctx, req.NamespacedName, &capp); err != nil {
 		if errors.IsNotFound(err) {
+			logger.Info(fmt.Sprintf("didn't find Capp: %s, from the namespace: %s", capp.Name, capp.Namespace))
 			return ctrl.Result{}, nil
 		}
-		return ctrl.Result{}, err
+		return ctrl.Result{}, fmt.Errorf("failed to get capp '%s': "+err.Error(), capp.Name)
```

Comment:
I don't think we need the `capp.Name` everywhere in the logs since it already exists in the `WithValues` of the logger.

We can change it to this so it's also more consistent with `rcs-ocm-deployer`:

```
fmt.Errorf("failed to get Capp: %s", err.Error())
```